site/zenodo_rdm/migrator/transform/records/metadata.py

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2023 CERN.
+#
+# ZenodoRDM is free software; you can redistribute it and/or modify it
+# under the terms of the MIT License; see LICENSE file for more details.
+
+"""Zenodo migrator metadata entry transformer."""
+
+from invenio_rdm_migrator.transform import Entry
+from nameparser import HumanName
+
+
+class ZenodoMetadataEntry(Entry):
+    """Metadata entry transform."""
+
+    @classmethod
```

Comment:
all methods could be classmethod or even staticmethod, should we change the interface? I do not see a point where there is class/instance state/attributes to be accessed since this are very strict hardcoded migration rules.